components/brave_rewards_ui/reducers/rewards_reducer.ts

```diff
@@ -28,6 +28,15 @@ const rewardsReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State
       state = { ...state }
       state.walletCreateFailed = true
       break
+    case types.GET_WALLET_PROPERTIES:
+      chrome.send('getWalletProperties', [])
+      break
+    case types.ON_WALLET_PROPERTIES:
```

Comment:
I see, so you're treating the response as an action to update the store? That seems like a reasonable route to go as well. I was thinking thinking something more like writing webui properties as `store.propertyname` and using a generic mechanism to write those to the actual store for the page, but either way works.

Reply:
so to basically have 1:1 mapping with redux store? this is how I did so far https://github.com/brave/brave-core/pull/293/files#diff-aa3505dbf36b5d03d8ba0751e0c99904R124 which I think it's quite the same as you were thinking.